Components/Optimizers/StandardStochasticGradientDescent/itkStochasticGradientDescentOptimizer.cxx

```diff
@@ -288,7 +288,7 @@ StochasticGradientDescentOptimizer::AdvanceOneStep()
     temp.t_Optimizer = this;
 
     /** Call multi-threaded AdvanceOneStep(). */
-    auto local_threader = ThreaderType::New();
+    auto local_threader = MultiThreaderBase::New();
```

Comment:
Might use thread pools, so might improve the performance of StochasticGradientDescentOptimizer 👍